addons/longhorn/1.1.2/install.sh

```diff
@@ -209,11 +209,6 @@ function maybe_init_hosts() {
                     printf "\n\t${GREEN}${prefix}tasks.sh | sudo bash -s longhorn-node-initilize${NC}\n\n"
                 fi
 
-                if ! prompts_can_prompt ; then
-                    logWarn "Install Longhorn prerequisites prompt explicitly ignored"
-                    return
-                fi
-
```

Comment:
I would prefer for this logic to still exist, but have prompts_can_prompt run once before starting the tee (and storing the result)

Does that seem plausible?

Reply:
HI @laverya,

Thank you a lot for looking it.  

I am not sure if I could properly follow up your suggestion. The logic `prompts_can_prompt ` seems to exist only because we were getting the input with `read PROMPT_RESULT < /dev/tty` instead `read -p "<string output>: " PROMPT_RESULT`.

So, why would we need to have prompts_can_prompt running before we starting the tee? 
What is the use case that would require it? What is the workflow the requires it?

Reply:
The idea behind `prompts_can_prompt` is to determine whether the script is being run in an interactive terminal where the user can respond to dynamic questions (things like "can we upgrade k8s" or "what is your IP address, I can't find it"). Some users run kURL as part of automation where this is not possible. (And so does testgrid)

If we can't figure out the IP and the user can't answer dynamically, we need to exit the script immediately instead of waiting forever for input.

A 'good' way to test this would be to run something like `sudo bash install.sh | tee` - currently IRC 'prompts_can_prompt' would return false in that scenario.

Reply:
In fact, I discovered that the problem was only the condition see: https://replicated.slack.com/archives/C0499EGNPNY/p1672836175693969?thread_ts=1672774404.239299&cid=C0499EGNPNY

So, I am closing it as outdated.